proto/tendermint/abci/types.proto

```diff
@@ -212,6 +211,12 @@ message ResponseCheckTx {
   repeated Event events     = 7
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
   string codespace = 8;
+  string sender    = 9;
```

Comment:
This doesn't feel like a rename :-) 

Is this bringing in some other changes to make the merge easier?

Reply:
This is the result of updating TM to 0.34.21. I'll create separate PR for this; I missed this while working on #528.